commands/system/help.js

```diff
@@ -9,74 +11,170 @@ class Help extends Command {
+            output += `\n\`${message.settings.prefix + c.help.name}\` | ${c.help.description.length > 80 ? `${c.help.description.slice(0, 80)}...` : c.help.description}`;
           }
           num++;
         }
       }
-    
+
       if (num) {
-        embed.setTitle("Command category help")
-          .setDescription(`A list of commands in the ${type} category.\n(Total of ${num} commands in this category)\n\nTo get help on a specific command do \`${message.settings.prefix}help <command>\`\n\n${num > 10 && pg === 1 ? `To view more commands do\` ${message.settings.prefix}help <category> 2\`` : "" }`)
-          .addField("Commands", output);
+        Math.ceil(num / perpage) === 1 ?
+          embed.setTitle(type.toProperCase())
+            .setDescription(`A list of commands in the ${type} category.\n(Total of ${num} commands in this category)\n\nTo get help on a specific command do \`${message.settings.prefix}help <command>\``)
+            .addField("Commands", output)
+          :
+          embed.setTitle(`Page ${page}/${Math.ceil(num / perpage)} for ${type.toProperCase()}`)
+            .setDescription(`A list of commands in the ${type} category.\n(Total of ${num} commands in this category)\n\nTo get help on a specific command do \`${message.settings.prefix}help <command>\``)
+            .addField("Commands", output);
       }
     }
 
-    if (this.client.commands.has(type) || this.client.commands.some(command => command.conf.aliases.includes(type))) {
-      const cm = this.client.commands.get(type) || this.client.commands.get(this.client.aliases.get(type));
+    if (this.client.commands.has(type) || this.client.aliases.has(type)) {
+      const cm = this.client.commands.get(type) || this.client.commands.get(this.client.aliases.get((type)));
       if (level < this.client.levelCache[cm.conf.permLevel]) return;
-      embed.setTitle(cm.help.name.toProperCase())
+      embed.setTitle(cm.help.name)
         .addField("Command description", cm.help.description)
         .addField("Command usage", `\`${cm.help.usage}\``)
-        .addField("Command aliases", cm.conf.aliases.length == 0 ? "None" : cm.conf.aliases.join(", ") );
+        .addField("Command aliases", cm.conf.aliases.length === 0 ? "None" : cm.conf.aliases.join(", "));
+    }
+    const msg2 = await message.channel.send(embed);
+    const totalpages = Math.ceil(num / perpage);
+    if (!message.guild.me.hasPermission(["MANAGE_MESSAGES"])) {
+      await message.channel.send("I don\t have permission to remove reactions, please do this manually.");
+    }
+
+    if (msg2.embeds[0].title) {
+      if (msg2.embeds[0].title.includes("Page")) {
+        for (const emoji of emojis) await msg2.react(emoji);
+      }
     }
-    
-    if (!embed.fields[0]) return;
 
-    return message.channel.send(embed);
+    const select = await msg2.createReactionCollector(
```

Comment:
[`Message#createReactionCollector()`](https://discord.js.org/#/docs/main/stable/class/Message?scrollTo=createReactionCollector) does not return a Promise, but a [`ReactionCollector`](https://discord.js.org/#/docs/main/stable/class/ReactionCollector) instance.